docs/0140.md

```diff
@@ -0,0 +1,113 @@
+# Rules for AEP-140: Field Names
+
+[aep-140]: https://aep.dev/140
+
+## aep-140-boolean-field-naming
+
+**Rule**: Boolean fields should omit the prefix "is".
+
+This rule enforces that all boolean properties don't start with the prefix
+"is".
+
+### Details
+
+This rule looks for "boolean" properties and complains if the 'boolean' field
+name starts with the prefix "is".
+
+### Examples
+
+**Incorrect** code for this rule:
+
+```yaml
+'schemas':
```

Comment:
the other examples don't quote the keys (similar to almost every other yaml example I've seen) - I think it'd be good to follow the existing style unless there's a reason not to.